src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -553,7 +554,8 @@ public void mergeAlignment(final File referenceFasta) {
      * No return value, modifies the provided record.
      */
     public static void fixNmMdAndUq(final SAMRecord record, final ReferenceSequenceFileWalker refSeqWalker, final boolean isBisulfiteSequence) {
-        final byte[] referenceBases = refSeqWalker.get(refSeqWalker.getSequenceDictionary().getSequenceIndex(record.getReferenceName())).getBases();
+        final ReferenceSequence refSeq = refSeqWalker.get(refSeqWalker.getSequenceDictionary().getSequenceIndex(record.getReferenceName()));
```

Comment:
The part I meant to extract was the index...`refSeqWalker.getSequenceDictionary().getSequenceIndex(record.getReferenceName())`